src/components/FolderModal.jsx

```diff
@@ -62,7 +65,20 @@ const FolderModal = ({ displayTitle, displayText, onClose, folderOrCategoryName,
     () => selectRenameApiCall(isCollection, siteName, folderOrCategoryName, subfolderName, newDirectoryName),
     {
       onError: () => errorToast(`There was a problem trying to rename this folder. ${DEFAULT_RETRY_MSG}`),
-      onSuccess: () => window.location.reload(),
+      onSuccess: () => {
+        if (!isCollection) {
+          // Resource folder
+          console.log('hey')
```

Comment:
to remove

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/389/commits/6ad1ba36be8bd15f142a8b692895dc71c0794648